### ISSUE TITLE
Doesn't show textures for all blocks

### ISSUE THREAD

**Issue by lastuniverse** [PrismarineJS/prismarine-web-client#184]
![image](https://user-images.githubusercontent.com/3473738/119243666-26180280-bb69-11eb-856f-c7d57067b040.png)

```
three.module.js:24647 Uncaught TypeError: material.onBuild is not a function
    at getProgram (three.module.js:24647)
    at setProgram (three.module.js:24809)
    at WebGLRenderer.renderBufferDirect (three.module.js:24010)
    at renderObject (three.module.js:24588)
    at renderObjects (three.module.js:24561)
    at WebGLRenderer.render (three.module.js:24341)
    at animate (index.js:253)
```

**Comment by AwesomestCode** [comment]
Well, this is embarrassing. My guess is that this was caused by the latest THREE.js upgrade. I don't know if that was tested by a human.

**Comment by AwesomestCode** [comment]
Issue confirmed (can reproduce). Will take a deeper look into this.

**Comment by ShrimpyStuff** [comment]
This was closed in Awesomest's pr

**Comment by AwesomestCode** [comment]
Closed by #185
